packages/actor-query-operation-orderby-sparqlee/lib/ActorQueryOperationOrderBySparqlee.ts

```diff
@@ -41,12 +42,15 @@ export class ActorQueryOperationOrderBySparqlee extends ActorQueryOperationTyped
     let { bindingsStream } = output;
 
     // Sorting backwards since the first one is the most important therefore should be ordered last.
+    const orderByEvaluator = await this.expressionEvaluatorFactory.createTermComparator({ context });
+
     for (let i = operation.expressions.length - 1; i >= 0; i--) {
       let expr = operation.expressions[i];
       const isAscending = this.isAscending(expr);
       expr = this.extractSortExpression(expr);
       // Transform the stream by annotating it with the expr result
-      const evaluator = this.expressionEvaluatorFactory.createEvaluator(expr, context);
+      const evaluator = (await this.expressionEvaluatorFactory
```

Comment:
Why not?

Reply:
The orderTypes expects an RDF.Term right now. Would you want it to receive a Binding just like evaluators do? So it would evaluate and then return a result?

Reply:
Nvm, I thought that object was not being used, but it is.